beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSource.java

```diff
@@ -72,7 +80,19 @@ public SafeFuture<Void> requestBlobsSidecarsByRange(
       return delegate.requestBlobsSidecarsByRange(startSlot, count, listener);
     } else {
       return asyncRunner.runAfterDelay(
-          () -> requestBlobsSidecarsByRange(startSlot, count, listener), Duration.ofSeconds(3));
+          () -> requestBlobsSidecarsByRange(startSlot, count, listener), Duration.ofSeconds(DELAY));
+    }
+  }
+
+  @Override
+  public SafeFuture<Void> requestBlobSidecarsByRange(
+      final UInt64 startSlot, final UInt64 count, final RpcResponseListener<BlobSidecar> listener) {
+    if (blobSidecarsRateTracker.wantToRequestObjects(count.longValue()) > 0) {
+      LOG.debug("Sending request for {} blob sidecars", count);
+      return delegate.requestBlobSidecarsByRange(startSlot, count, listener);
+    } else {
+      return asyncRunner.runAfterDelay(
+          () -> requestBlobSidecarsByRange(startSlot, count, listener), Duration.ofSeconds(DELAY));
```

Comment:
shouldn't we decrease count here to `blobSidecarsRateTracker.wantToRequestObjects(count.longValue())` if it's smaller? I see that in blocks we ignore it, but it sounds as good behavior from my side

Reply:
As discussed, I created a separate issue for the requests throttling: https://github.com/ConsenSys/teku/issues/6874